glib/src/param_spec.rs

```diff
@@ -1370,6 +1370,77 @@ impl ParamSpecVariant {
     }
 }
 
+pub trait Parametrize {
+    type Spec;
+    fn get(&self) -> crate::Value;
+}
+pub trait ParametrizeMut {
+    fn set(&self, value: &crate::Value);
```

Comment:
Why the `set` / `get` methods here? You could simply require that all the types implement `ValueType` and then you get all that for free :)

Reply:
I guess because you also implement that on top of wrapper types like `RefCell`. I think this should be split into two traits somehow. One for the underlying values, and another for the storage

Reply:
Makes sense